queries/orderbook/batch_rewards.sql

```diff
@@ -0,0 +1,97 @@
+WITH observed_settlements AS (SELECT
+                                -- settlement
+                                tx_hash,
+                                solver,
+                                s.block_number,
+                                -- settlement_observations
+                                effective_gas_price * gas_used AS execution_cost,
+                                surplus,
+                                fee,
+                                -- auction_transaction
+                                at.auction_id
```

Comment:
Nit: I think it would make the query more understandable to add the table from where each column is coming.

`tx_hash` -> `s.tx_hash`